composer.json

```diff
@@ -13,7 +13,7 @@
 		"wmde/fun-validators": "~3.0.0"
 	},
 	"require-dev": {
-		"phpunit/phpunit": "~9.2",
+		"phpunit/phpunit": "~9.2.6",
```

Comment:
This is not a downgrade

Reply:
It is when your local composer updates it to >9.3

Reply:
Huh yeah. What what I thinking :D